offsets/src/analysis/misc.rs

```diff
@@ -68,8 +68,9 @@ fn local_entity_handle(f: &mut super::Output, bin: PeFile<'_>) {
 fn local_player(f: &mut super::Output, bin: PeFile<'_>) {
 	// The global instance of C_GameMovement contains as its first member a pointer to local player right after its vtable.
 	let mut save = [0; 4];
-	if bin.scanner().finds_code(pat!("8981??0000 488B1D${'}"), &mut save) {
-		let local_player_ptr = save[1];
+	// old: pat!("8981??0000 488B1D${'}")
+	if bin.scanner().finds_code(pat!("488B05u4' 488D0D???? 4488????? 4C89"), &mut save) {
```

Comment:
Replace `u4'` with `${'}`. $ is a symbol that follows these RIP relative addresses and {} just temporarily switches, captures the address with ' and returns to let the rest of the pattern match.

Reply:
Thank you very much, I enjoy improving my code through code reviews.